### DISCUSSION TITLE
Documentation missing

### DISCUSSION THREAD

**Discussion by AndreaChavezMunoz** [mooch443/trex#97] [Questions]
Hi,
I am just learning how to use your program and I was trying to use the live tracking feature. I don't know if this is a silly question, but I can figure out how to connect a normal webcam. The documentation only mentions how to work with a balser camera. What is the command to use an external camera?
Thank you

**Comment by mooch443** [comment]
Hey, this is definitely not a silly question! I will add this to the upcoming documentation enhancements project.

The way to do this is to change the input via ``-i webcam`` and then enable closed loop using [`enable_closed_loop`](https://trex.run/docs/parameters_tgrabs.html#enable_closed_loop) (or just [`enable_live_tracking`](https://trex.run/docs/parameters_tgrabs.html#enable_live_tracking) if you do not actually need live feedback. `enable_closed_loop` implies this option). The main difference between the two options is that `enable_closed_loop` is allowed to drop frames if required, and also starts up a python subprocess where you can script responses to individuals' positions/body shapes/visual fields. Live tracking (without closed loop) will keep all frames in memory, or slow down the recording/conversion if needed.
